src/openforms/forms/api/serializers/logic/action_serializers.py

```diff
@@ -103,19 +102,16 @@ class LogicComponentActionSerializer(serializers.Serializer):
             )
         ),
     )
-    form_step = URLRelatedField(
+    form_step = ActionFormStepUUIDField(
```

Comment:
I guess this technically is a breaking change in the API - maybe we should keep the old field and deprecate it, and add a new field `form_step_uuid`?